src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -150,6 +150,9 @@
     "span.group": PREFIX + 252,
     "transaction.method": PREFIX + 253,
     "span.category": PREFIX + 254,
+    "span.contribution": PREFIX + 255,
```

Comment:
hmm, actually @nikhars left this comment on my data requests [doc](https://www.notion.so/sentry/span-contribution-9b5a0979a81645789db6877593729cf4?pvs=4)
> Tag values are always stored as strings in the metrics platform. Also, we should avoid having multiple values in a single tag key. So I would second Abhi’s suggestion to have different tag key like is_ttid and is_ttfd

So should we not add this for now?

Reply:
OK, removed for now.